CODEWALK.md

```diff
@@ -636,3 +640,15 @@ Our implementation of [plumbing and porcelain](#plumbing-and-porcelain) embraces
 
 This idiom is unfortunately hidden away in a [wiki page about code review](https://github.com/golang/go/wiki/CodeReviewComments#interfaces).
 See also Dave Cheney on [SOLID Go Design](https://dave.cheney.net/2016/08/20/solid-go-design).
+
+### Observability
+
+go-filecoin uses [Opencensus-go](https://github.com/census-instrumentation/opencensus-go) for stats collection and distributed tracing instrumentation. Stats are exported for consumption via [Prometheus](https://prometheus.io/) and traces are exported for consumption via [Jaeger](https://www.jaegertracing.io/docs/1.11/).
+
+#### Metrics
+
+go-filecoin can be configured to collect and export metrics to Prometheus via the `MetricsConfig`. The details of this can be found inside the [`config/`](https://github.com/filecoin-project/go-filecoin/blob/ce1f739b1569f95279a15a6a4e9978186c680d38/config/config.go) package. To view metrics from your filecoin node using the default configuration options set the `prometheusEnabled` value to `true`, start the filecoin daemon, then visit `localhost:9400/metrics` in your web-browser. 
+
+#### Tracing
+
+go-filecoin can be configured to collect and export traces to Jaeger via the `TraceingConfig`. The details of this can be found inside the [`config/`](https://github.com/filecoin-project/go-filecoin/blob/ce1f739b1569f95279a15a6a4e9978186c680d38/config/config.go) package. To collect traces from your filecoin node using the default configuration options set the `jaegerTracingEnabled` value to `true`, start the filecoin daemon, then follow the [Jaeger Getting](https://www.jaegertracing.io/docs/1.11/getting-started/#all-in-one) started guide.
```

Comment:
nit: `TraceingConfig` should be `TraceConfig` and the link provided does not have this implemented yet.